MeetupPlatformApi/Migrations/20220401115515_MeetupPKFromIntToGuid.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class MeetupPKFromIntToGuid : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropPrimaryKey(
+                name: "pk_meetups",
+                table: "meetups");
+
+            migrationBuilder.DropColumn(
+                name: "id",
+                table: "meetups");
+
+            migrationBuilder.AddColumn<Guid>(
+                name: "id", 
+                table: "meetups",
+                nullable: false,
+                defaultValue: Guid.Empty);
+
+            migrationBuilder.AddPrimaryKey(
+                name: "pk_meetups",
+                table: "meetups",
+                column: "id");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<int>(
```

Comment:
This will break for at least 3 reasons:
1. We don't have table `Meetups`
2. We don't have column named `Id` on our `meetups` table
3. PostgreSQL can't magically cast `uuid` to `int`

Reply:
Fixed.